gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -223,7 +232,7 @@ protected PublisherSettings(MethodBuilders methods) {
   }
 
   /**
-   * Returns the ApiCallableBuilder for the API method createTopic.
+   * Returns the builder for the API method createTopic.
```

Comment:
I don't think you mentioned that yet. I'll go ahead and make that change in GAX. Would you like to see that in this PR, or is a following PR ok?